docs/docs/usage/scaladoc/staticSite.md

```diff
@@ -99,23 +106,44 @@ Scaladoc by default uses layout of files in `docs` directory to create table of
 ```yaml
 sidebar:
     - title: Blog
-    - title: Docs
-      url: docs/index.html
-    - title: Usage
+    - title: My title
+      page: my-page1.md
+    - page: my-page2.md
+    - page: my-page3/subsection
+    - title: Reference
       subsection:
-        - title: Dottydoc
-          url: docs/usage/dottydoc.html
-        - title: sbt-projects
-          url: docs/usage/sbt-projects.html
+        - page: my-page3.md
+    - index: my-page4/index.md
+      subsection:
+        - page: my-page4/my-page4.md
+    - title: My subsection
+      index: my-page5/index.md
+      subsection:
+        - page: my-page5/my-page5.md
+    - index: my-page6/index.md
+      subsection:
+        - index: my-page6/my-page6/index.md
+          subsection:
+            - page: my-page6/my-page6/my-page6.md
 ```
 
-The `sidebar` key is mandatory, as well as `title` for each element. The
-default table of contents allows you to have subsections - albeit the current
-depth limit is 2 however it accepts both files and directories and latter can be used to provide deeper structures.
+The `sidebar` key is mandatory.
+On each level you can have three different types of entries: `page`, `blog` or `subsection`.
+
+`Page` is a leaf of the structure and accepts following attributes:
+- `title` \[optional\] - title of the page
```

Comment:
I suggest using parentheses instead of square brackets.

Reply:
Done